src/corona_hakab_model/main.py

```diff
@@ -14,30 +16,58 @@
 from supervisor import LambdaValueSupervisable, Supervisable, SimulationProgression
 
 
-
 from typing import TYPE_CHECKING
 if TYPE_CHECKING:
     import pandas as pd
 
+logger = logging.getLogger('application')
+logger.setLevel("Info")
```

Comment:
switch from "Info" to logging.INFO ("Info" gives me error when i run it)

Reply:
Done, I fixed it but overrode my own changes in a rebase